docs/developers.rst

```diff
@@ -22,13 +22,13 @@ http://sphinx.pocoo.org/). To make the documentation run the following command
 in the root directory::
 
   pip install sphinx
-  sphinx-build -M html docs/ docs/_build/
+  sphinx-build docs/ docs/_build/
 
 Packaging a Distribution
```

Comment:
I think this section is pointless and we should replace it with "publishing a release" and indicate that in order to publish to pypi, follow instructions in https://flit.pypa.io/en/stable/

 (pending more details).